manifests/init.pp

```diff
@@ -67,28 +75,30 @@
     ensure => 'installed',
   }
 
-  if $manage_init_script == true {
+  # Don't manage init script for Ubuntu
+  if ($manage_init_script == true) and ($::osfamily != 'Ubuntu') {
     file { '/etc/init.d/tac_plus':
       ensure  => 'file',
       content => template($init_template),
       owner   => 'root',
       group   => 'root',
       mode    => '0744',
-      before  => Service['tac_plus'],
+      before  => Service[$tac_plus_service],
     }
   }
 
   # TODO: what about the mode?
-  file { '/etc/tac_plus.conf':
+  file { $tac_plus_config:
     ensure  => 'file',
     content => template($tac_plus_template_real),
     owner   => 'root',
     group   => 'root',
     require => Package[$tacplus_pkg],
-    notify  => Service['tac_plus'],
+    notify  => Service[$tac_plus_service],
   }
 
-  if $manage_pam == true {
+  # PAM not supported on Ubuntu
+  if ($manage_pam == true) and ($::osfamily != 'Ubuntu') {
```

Comment:
Should also fail loud if ```$manager_pam``` is set to true on Ubuntu. Same loginc as for ```$manage_init_script``` should be applied here too.